src/components/Sidebar.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { SidebarContent, SidebarItem } from './Sidebar.style';
+export default function Sidebar({ onSelectCategory, categories }) {
+  const selectCategoryHandler = (categoryId) => {
+    onSelectCategory(categoryId);
+  };
+
+  return (
+    <SidebarContent>
+      <SidebarItem>
+        <h3>
+          <i>Categories</i>
+        </h3>
+      </SidebarItem>
+      {categories.map(({ id, isSelected, data: { name } }) => {
+        return (
+          <SidebarItem key={id} onClick={() => selectCategoryHandler(id)}>
+            <input type="checkbox" readOnly={true} checked={isSelected} />{' '}
+            {name}
```

Comment:
Creo que ese espacio extra que tienes solo para agregar un espacio podría bien estar contenido en las llaves del name. Se me hace más limpio, es todo.

```suggestion
            <input type="checkbox" readOnly={true} checked={isSelected} />
            {` ${name}`}
```

Reply:
Se cambia el espacio adentro de las llaves del name